types/knex-cleaner/knex-cleaner-tests.ts

```diff
@@ -0,0 +1,6 @@
+import { clean } from 'knex-cleaner';
+import * as Knex from 'knex';
+
+const knex: Knex = Knex({ dialect: 'pg' });
+
+clean(knex).then(console.log);
```

Comment:
This could use more cases, especially with the introduced options, maybe something along the lines of this?
```
clean(knex); // $ExpectType Promise<void>
clean(knex, {}); // $ExpectType Promise<void>
clean(knex, {mode: 'delete', restartIdentity: false, ignoreTables: ['table1', 'table2']}); // $ExpectType Promise<void>
clean(knex, {unknown: 1}); // $ExpectError
```

Reply:
I used your examples + all permutations of `KnexCleanerOptions` keys